### PR TITLE
Finish implementation of getAddressesFromBondTable

### PR DESCRIPTION
**Pull request by andresag01**
Finish the implementation of the new function getAddressesFromBondTable() in
SecurityManager that returns a table with a list of addresses from the peers
in the bond table.
@pan-

### CODE REVIEW

**Comment by andresag01** [comment]
**NOTE:** This PR requires changes to nrf51-sdk to expose the ah() function to calculate a hash.

File: source/nRF5xSecurityManager.h [hunk 1]
```diff
@@ -47,6 +47,73 @@ class nRF5xSecurityManager : public SecurityManager
...
+     *
+     * @return
+     *           BLE_ERROR_NONE if successful.
+     */
+    virtual ble_error_t getAddressesFromBondTable(Gap::Whitelist_t &addresses) const {
+        uint8_t i;
+
+        ble_gap_whitelist_t  whitelistFromBondTable;
+        ble_gap_addr_t      *addressPtr[YOTTA_CFG_WHITELIST_MAX_SIZE];
+        ble_gap_irk_t       *irkPtr[YOTTA_CFG_IRK_TABLE_MAX_SIZE];
```
Comment (andresag01) [inline source/nRF5xSecurityManager.h]: We have to tell the DeviceManager the maximum number of entries that we would like to fetch. I am not sure whether using the YOTTA defined size could be a limitation at some point.

**Comment by andresag01** [comment]
**NOTE:** If the bond table contains both a static address and an IRK for the same device, then the returned address list from  `getAddressesFromBondTable()` will contain two addresses for the same device i.e. one private resolvable and one static address.

File: source/nRF5xSecurityManager.h (hunk 1 above)
Comment (pan-) [inline source/nRF5xSecurityManager.h]: I don't think it will be a problem

**Comment by pan-** [comment]
All tests pass, I merge

**Pull request merged**